src/patternfly/components/InputGroup/input-group.scss

```diff
@@ -87,17 +84,23 @@
   textarea {
     min-height: var(--pf-c-input-group__textarea--MinHeight);
   }
+}
 
-  .pf-c-input-group__text {
-    display: flex;
-    align-items: center;
-    padding-right: var(--pf-c-input-group__text--PaddingRight);
-    padding-left: var(--pf-c-input-group__text--PaddingLeft);
-    font-size: var(--pf-c-input-group__text--FontSize);
-    color: var(--pf-c-input-group__text--Color);
-    text-align: center;
-    background-color: var(--pf-c-input-group__text--BackgroundColor);
-    border: var(--pf-c-input-group__text--BorderWidth) solid;
-    border-color: var(--pf-c-input-group__text--BorderTopColor) var(--pf-c-input-group__text--BorderRightColor) var(--pf-c-input-group__text--BorderBottomColor) var(--pf-c-input-group__text--BorderLeftColor);
-  }
+/* stylelint-disable */
+label.pf-c-input-group__text {
+  cursor: pointer;
+}
+/* stylelint-enable */
+
+.pf-c-input-group__text {
+  display: flex;
+  align-items: center;
+  padding-right: var(--pf-c-input-group__text--PaddingRight);
+  padding-left: var(--pf-c-input-group__text--PaddingLeft);
+  font-size: var(--pf-c-input-group__text--FontSize);
+  color: var(--pf-c-input-group__text--Color);
+  text-align: center;
+  background-color: var(--pf-c-input-group__text--BackgroundColor);
+  border: var(--pf-c-input-group__text--BorderWidth) solid;
+  border-color: var(--pf-c-input-group__text--BorderTopColor) var(--pf-c-input-group__text--BorderRightColor) var(--pf-c-input-group__text--BorderBottomColor) var(--pf-c-input-group__text--BorderLeftColor);
```

Comment:
You can add this in the `.pf-c-input-group__text` selector to support `label.pf-c-input-group__text`

```
  @at-root label#{&} {
    cursor: pointer;
  }
```

Reply:
Learning something new everyday!

Thanks, I'll fix that up :)